README.md

```diff
@@ -341,6 +342,7 @@ Useful Articles for developing on the micro:bit.
 - [micro:bit Radio Packets](https://ukbaz.github.io/howto/ubit_radio.html) - Explanation of the MakeCode radio packet specification (built on top of the micro:bit runtime specification) and how to communicate between MakeCode and MicroPython programs via radio.
 - [Synchronized Music on micro:bits](https://blog.flowblok.id.au/2018-02/synchronized-music-on-microbits.html) - Building a micro:bit mesh network so they can play music synchronized across a large area.
 - [Using the Built-in Sensors](https://learn.adafruit.com/micro-bit-lesson-1-using-the-built-in-sensors) - Learn how to use the micro:bit's built-in accelerometer and magnetometer.
+- [Read micro:bit data from Linux via Bluetooth (BLE)] - Random notes and examples about micro:bit BLE and Linux. 
```

Comment:
Could you submit this entry a separate PR? That way there is only one entry per PR and is a lot easier for people to check out the new additions.